packages/web-app-files/src/components/SideBar/Versions/FileVersions.vue

```diff
@@ -139,3 +148,56 @@ export default defineComponent({
   }
 })
 </script>
+
+<style lang="scss" scoped>
+#oc-file-versions-sidebar {
+  > ul {
+    list-style: none;
+
+    .spacer {
+      border-left: 1px solid var(--oc-color-border);
+      margin-left: calc(-1 * var(--oc-space-large)) !important;
+    }
+
+    > li.version-item {
+      border-left: 1px solid var(--oc-color-border);
+      margin-left: calc(-1 * var(--oc-space-large)) !important;
+      padding-left: var(--oc-space-medium);
+      padding-bottom: var(--oc-space-medium);
+      margin-top: calc(-1 * var(--oc-space-small));
+
+      &::before {
+        content: '';
+        display: block;
+        width: 11px;
+        height: 11px;
+        position: absolute;
+        left: -6px;
+        top: 4px;
+        background-color: var(--oc-color-border);
+        border-radius: 50%;
+      }
+      .version-details {
+        .version-date {
+          font-weight: 600; // No css class for font-weight?
```

Comment:
feel free to add it in `packages/design-system/src/tokens/ods/font.yaml`. :-) You could add a `weight` section and then add some reasonable values.

Reply:
if you don't want to add the weight var, just delete the comment, fine as well

Reply:
ok added. how do I compile the yml file?

Reply:
pnpm build:tokens

that’s also part of pnpm build

Reply:
done.
How would one now programmatically create corresponding css classes? Like for spacing:
```
@each $sizeKey, $sizeValue in $sizes {
  &-#{$sizeKey} {
    #{$directiveValue}-#{$direction}: $sizeValue !important;
  }
}
```